tokens/WWW-900614/info.json

```diff
@@ -0,0 +1,11 @@
+{
+"website": "https://www.worldwebwhisky.com/",
+"description": "WorldWebWhisky is the first decentralised spirits marketplace",  "ledgerSignature": "",
```

Comment:
please add `"ledgerSignature": "",` on a new line - or you might as well remove it for now

Reply:
works now?